build/lib/accounting/write_jobs.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+
+###############################################################
+# Copyright 2020 Lawrence Livermore National Security, LLC
+# (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+#
+# This file is part of the Flux resource manager framework.
+# For details, see https://github.com/flux-framework.
+#
+# SPDX-License-Identifier: LGPL-3.0
+###############################################################
+from __future__ import print_function
+
+import sys
+import sqlite3
+import pandas as pd
+import time
+
+import flux.job
+import flux.constants
+import flux.util
+from flux.core.inner import raw
+
+
+def write_to_db(inactive):
+    # open connection to database
+    print("Opening JobCompletion DB...")
+    conn = sqlite3.connect("JobCompletion.db")
+    print("Opened JobCompletion DB successfully\n")
+
+    # insert values into DB one at a time by getting the values from
+    # each job completion tuple
+    print("Inserting values into DB...\n")
+    for job in inactive:
+        conn.execute(
+            """
+            INSERT OR IGNORE INTO inactive
+            VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);
+            """,
+            (
+                job["t_submit"],
+                job["name"],
+                job["t_run"],
+                job["t_cleanup"],
+                job["userid"],
+                job["ntasks"],
+                job["t_inactive"],
+                job["t_depend"],
+                job["priority"],
+                job["state"],
+                job["t_sched"],
+                job["id"],
+            ),
+        )
+        print("insertion into inactive table successful")
+
+    # commit changes
+    conn.commit()
+
+    # make sure the writes to DB were successful and we can
+    # retrieve them
+    sel_statement = "SELECT * FROM inactive"
+    print(pd.read_sql_query(sel_statement, conn))
```

Comment:
Is the plan going forward to use pandas to execute the sql queries and interact with the results?  That actually seems totally reasonable to me.  Working with the returned values via Pandas will be much faster than with Python directly.  If that is the plan, the Pandas requirement should be documented somewhere and included in the `requirements.txt` (which looks empty right now).

If this is just a convenience thing for debugging, and you plan to stick with the Python standard library for the main code, I'd wrap this print up in a conditional import:
```suggestion
    try:
        import pandas as pd
        print(pd.read_sql_query(sel_statement, conn))
    except ImportError:
        print("Failed to import pandas, skipping debug print of results")
```

Reply:
Also note, that if you want to use pandas to drive this script, we'll need to settle on a minimum support version.  Pandas recently tagged 1.0 (which I think has some breaking changes from 0.23), but I doubt that is available on TOSS4 (which I think is built on top of RHEL 8......I've lost track).

Reply:
I am planning on using pandas - it provides a lot of nice formatting options with SQLite queries along with executing queries. Right now, I am using `0.25.3`. I should have added that to the requirements page, thank you for pointing that out!